tests/test_invenio_assets_filters.py

```diff
@@ -129,5 +129,6 @@ def test_angular_gettext(self, static_dir):
                                    output='catalog.js')
             bundle.build()
             catalog = self.get('catalog.js')
+            assert '"fr"' in catalog
```

Comment:
I wanted to make sure that the `"fr"` is there. Otherwise I would need to write a regex since `fr` can be a part of the text anyway.